packages/application-generic/src/services/in-memory-provider/redis-provider.ts

```diff
@@ -37,17 +39,19 @@ export interface IRedisProviderConfig {
 
 export const getRedisProviderConfig = (): IRedisProviderConfig => {
   const redisConfig: IRedisConfig = {
-    host: process.env.REDIS_CACHE_SERVICE_HOST,
-    port: process.env.REDIS_CACHE_SERVICE_PORT || String(6379),
+    db: process.env.REDIS_DB_INDEX,
+    host: process.env.REDIS_HOST || 'localhost',
+    port: process.env.REDIS_PORT || String(6379),
```

Comment:
Everything started here when I noticed that I had set for the Redis instance provider for InMemoryProviderService the wrong settings when trying to hook up the BullMqService to InMemoryProviderService in the WS app in the MemoryDB task.
Fixing it here already.